src/Controllers/memberController.js

```diff
@@ -0,0 +1,69 @@
+// const pool = require('../Database/DBconnection');
+// const jwt = require('jsonwebtoken');
+const Sequelize = require('sequelize');
+const MemberRespository = require('../Models/memberModel');
+const TrainerRespository = require('../Models/trainerModel');
+
+//--------------------------------- GET ----------------------------------//
+
+function findAll(req, res) {
+    MemberRespository.findAll().then((result) => res.json(result));
+}
+
+//---------------------------------- POST -------------------------------//
+
+async function addMember(req, res) {
```

Comment:
Para cada operação do CRUD que voce for fazer é sempre interessante usar o bloco try catch para tratar erros inesperados. Fica assim: ```
try {
await MemberRespository.create(
        {
            member_name: req.body.member_name,
            member_email: req.body.member_email,
            password: req.body.password,
            trainer_id: req.params.id
        }, 
        {
            where: {
                trainer_id: req.params.id,
            }
        }
    ).then((result) => res.json(result));
    TrainerRespository.findByPk(req.params.id)
} catch (error) {
return res.status(500).send({
        "error message": "Internal Server Error",
      });
```